templates/accuracy_notice_email.html

```diff
@@ -34,33 +46,5 @@
   </p>
 </section>
 
-
-<section id="details" style="{{styles.section}}">
-  <p>Your feature is slated to launch soon for m{{milestone}}:</p>
-  <p>
-    {% include "estimated-milestones-table.html" %}
-  </p>
-  <p>
-    Your feature entry is an important resource for
-    cross functional teams that help drive adoption of new features and
-    enterprise IT admins who might be affected by web platform changes.
-  </p>
-
-  <p>
-    We need to know whether your plans are changing or staying
```

Comment:
How about "The Blink launch process needs to know" instead of "We need to know"? It's not just our team that needs this information, as we're constantly reminded by others, and this framing could add to the sense of urgency and legitimacy that the notification should have.

Reply:
Addressed the comment. The email subject prefix [webstatus] comes from somewhere else. Let me chat with Jason tomorrow

Reply:
I still prefer "we".  It's the simplest way to write it, which allows the emphasis to be on the action that is needed.

Reply:
Agree with "we" here. It also sounds more personal, which always works better in a request